TROUBLESHOOTING.md

```diff
@@ -1,6 +1,8 @@
-# podman
+# Troubleshooting
```

Comment:
There's no need to add this an extra first level. This is already the troubleshooting document.